docs/developer-guide/build-zone-setup-triton.md

```diff
@@ -178,8 +178,15 @@ So, the two ways of creating build zones are:
  * Choose a CN to provision to (selecting "headnode" will ensure the build zone
    is provisioned on that headnode)
  * Choose the "external" network
- * Add the following Customer metadata, `{"user-script": "/usr/sbin/mdata-get root_authorized_keys > ~root/.ssh/authorized_keys ; /usr/sbin/mdata-get root_authorized_keys > ~admin/.ssh/authorized_keys; svcadm enable manifest-import"}`
+ * Add the following Customer metadata, `{"user-script": "/usr/sbin/mdata-get root_authorized_keys > ~root/.ssh/authorized_keys ; /usr/sbin/mdata-get root_authorized_keys > ~admin/.ssh/authorized_keys; svcadm enable manifest-import"}` along with supplying your public ssh key.
```

Comment:
Could we say exactly where the public ssh key should be added, perhaps as a separate list item?

Could we also move the customer metadata section into a code-block rather than inlining it in the list of things to add?